scipy/stats/_continuous_distns.py

```diff
@@ -821,6 +821,10 @@ def fit(self, data, *args, **kwds):
 
         return a, b, floc, fscale
 
+    def _entropy(self, a, b):
+        return sc.betaln(a, b) - (a - 1) * sc.psi(a) - \
+           (b - 1) * sc.psi(b) + (a + b - 2) * sc.psi(a + b)
```

Comment:
Here and below: as @mdhaber pointed out, PEP 8 says to prefer implied line continuation inside parentheses instead of using the continuation character `\`.